d2l-button-behavior.js

```diff
@@ -97,7 +97,8 @@ D2L.PolymerBehaviors.Button.Behavior = {
 		 */
 		type: {
 			type: String,
-			reflectToAttribute: true
+			reflectToAttribute: true,
+			value: 'button'
```

Comment:
I was thinking about this change as well, as it eliminate some of the change in your LMS PR.  I was hesitant to suggest this because the this is not consistent with the browser default for buttons, and should also be released with a major version bump.  On the other hand, it's almost always what we want.  So, I sort of have mixed thoughts on it, and don't feel super strongly either way.

Reply:
Yeah, I felt similarly. I justified it by:

1) I think the browser default is dumb
2) For most of our button types (icon, subtle, even normal) we probably rarely have them submitting forms anyway.

For the major version thing -- agreed -- but were hoping we could lump this in with the existing major version bump to Polymer 3. A little bit sketchy, but hopefully OK?

Reply:
Yeah, things ought to be tested as part of that, so hopefully we'd catch any cases that were depending on it doing a submit.